prow/jobs/custom/label-sync.yaml

```diff
@@ -36,25 +36,34 @@ presubmits:
         # Set --confirm=false to only validate the configuration file.
         - --confirm=false
         - --orgs=knative-sandbox,knative
-        - --github-app-id=$(GITHUB_APP_ID)
-        - --github-app-private-key-path=/etc/github/cert
-        - "--github-hourly-tokens=2000"
-        - "--github-allowed-burst=1000"
+        # Below had to be comment out when we moved labels.yaml to community.
```

Comment:
This issue isn't related to the repo move. It is due to the change I made in https://github.com/knative/test-infra/pull/3198, and it only happens to the presubmit Prow job when `--confirm=false`.